authentication/oidc.go

```diff
@@ -59,8 +60,14 @@ func NewOIDC(logger log.Logger, configs []TenantOIDCConfig) (http.Handler, map[s
 		}
 
 		r := chi.NewRouter()
-		r.Handle("/login", p.LoginHandler())
-		r.Handle("/callback", p.CallbackHandler())
+
+		const loginRoute = "/login"
+
+		r.Handle(loginRoute, otelhttp.WithRouteTag(prefix+loginRoute, p.LoginHandler()))
+
```

Comment:
let's not add to many new lines, it's not helping  - usually we do them to separate differences

Reply:
The linter complains everytime I remove those lines with `expressions should not be cuddled with declarations or returns (wsl)` :sweat_smile: